docs/content/preview/yugabyte-platform/alerts-monitoring/prometheus-custom/_index.md

```diff
@@ -25,3 +25,12 @@ You can do this in the following ways:
 - [Scrape directly from universe nodes](prometheus-scrape/)
 
     Your independent Prometheus instance can scrape data from database nodes directly (in the case of VM-based universes) or scrape data from a Kubernetes Prometheus Operator Service Monitor (in the case of K8s-based universes). This data scraping runs in parallel to and independently from the YugabyteDB Anywhere-embedded Prometheus.
+
```

Comment:
This URL should be something like

curl -k --request GET   --url https://portal.dev.yugabyte.com/api/v1/grafana_dashboard   --header 'Accept: application/json'  --header 'X-AUTH-YW-API-TOKEN: 123'